### DISCUSSION TITLE
Can import at local (not use UPM)

### DISCUSSION THREAD

**Discussion by wnsxor1993** [Voltstro-Studios/UnityWebBrowser#253] [Q&A]
I'm curious about whether there is a way to use UWB without using UPM. 
I couldn't find any information on this on the related pages. I want to build for mobile as a target without removing the package, and to do that, I need to import the package at local. 
I need this for adding preprocessor directives in the PostprocessBuild section of the engine-related scripts to enable building for mobile while keeping the package imported. 
(This doesn't mean using UWB on mobile but simply being able to build for mobile without removing the package.)

**Comment by jcs090218** [comment]
The answer is no. But I do hope there is a `.unitypackage` file.
